src/test_l1_tvSettings.c

```diff
@@ -307,13 +307,12 @@ void test_l1_tvSettings_negative_TvTerm (void)
  * | Variation / Step | Description | Test Data | Expected Result | Notes |
  * | :-------: | ------------- | --------- | --------------- | ----- |
  * | 01 | call TvInit() -  Initialise and get a valid instance of the TV client | void | tvERROR_NONE | Should Pass |
- * | 02 | Call RegisterVideoFormatChangeCB() - Register with the obtained data handle and a valid callback function | tvVideoFormatCallbackData& | void | tvERROR_NONE | Should Pass |
+ * | 02 | Call RegisterVideoFormatChangeCB() - Register with the obtained data handle and a valid callback function | tvVideoFormatCallbackData * | void | tvERROR_NONE | Should Pass |
```

Comment:
Remove void, already test data is added as tvVideoFormatCallbackData *

Reply:
Done